documentation.json

```diff
@@ -336,7 +341,12 @@
         "command": "confirm",
         "parameters": ["token"],
         "description": "Send to KoalaBot in dms to confirm the verification of an email"
-      }
+      },
+      {
+	"command": "verifyDM",
+	"parameters": ["toggle"]
+	"description" "Toggle the verify DM for a guild, that sends the user a list of emails to verify"
```

Comment:
please make sure the colon is there between "description" and "Toggle ...", also need to add back in the comma on the line above (347)

Reply:
Done